src/define_config.ts

```diff
@@ -148,4 +150,22 @@ export const stores: {
       }
     })
   },
+  dynamodb: (config) => {
+    return configProvider.create(async () => {
+      const { DynamoDBStore } = await import('./stores/dynamodb.js')
+      const { DynamoDBClient } = await import('@aws-sdk/client-dynamodb')
+      const client = new DynamoDBClient(config?.clientConfig ?? {})
+
+      return (_, sessionConfig: SessionConfig) => {
+        return new DynamoDBStore(
+          client,
+          config.tableName,
+          sessionConfig.age,
+          config.keyAttribute,
+          config.valueAttribute,
+          config.expiresAtAttribute
```

Comment:
Are there any legit reasons for allowing to customize the attribute names? 

I get it provides flexibility, but if the names are hardcoded, do we have any downsides?

Reply:
@thetutlage 
Since the `keyAttribute` must be defined when creating a DynamoDB table, flexibility is necessary to allow it to be overwritten later.
As for the `valueAttribute` and `expiresAtAttribute`, to be honest, I can’t think of many cases where hardcoding them would cause issues.
I referred to [Laravel’s implementation](https://github.com/laravel/framework/blob/11.x/src/Illuminate/Cache/DynamoDbStore.php#L71), and perhaps there may be cases where you want to share the session table with an already existing application.

Reply:
Sure. So let's hardcode `valueAttribute` and the `expiresAtAttribute` attribute and let's see how it goes. We can always make them attributes configurable later.

Reply:
@thetutlage 
I have hardcoded the `valueAttribute` and `expiresAtAttribute`. I have also updated [the documentation](https://github.com/adonisjs/v6-docs/pull/148/commits/d36ff1465889b3ff299135bb6c9616f20f9764b1).